contracts/Certificate.sol

```diff
@@ -73,6 +73,15 @@ contract Certificate is
    */
   mapping(uint256 => uint256) private _purchaseAmounts;
 
+  /**
+   * @notice Keeps track of any discrepancy between the total number of NRTs guaranteed by this contract and the
+   * number of NRTs currently held. In other words, the number of NRTs held minus the number of NRTs guaranteed.
+   * @dev This is used to provide a redundant, transparent account of the number of NRTs that may still need to be
+   * replaced in the case of released removals. This number should only be non-zero if removals are in the process of
+   * being replaced.
+   */
+  int256 private _guaranteeDiscrepancy;
```

Comment:
WRT to what's below, this should never be above 0 (meaning there should never be more replaced removals than have been released. Given that, we might be able to name this in a clearer way (IE - `numberOfRemovalsReleased`).

Reply:
IRL decision, we're going with `nrtDeficit`